frontend/src/views/WheresMyGene/components/HeatMap/components/Chart/style.ts

```diff
@@ -1,4 +1,6 @@
 import { css } from "@emotion/css";
+/* eslint-disable @typescript-eslint/ban-ts-comment --- TODO revisit lint error below */
+// @ts-ignore - TODO revisit lint error
```

Comment:
ah I can't reproduce the lint error locally. What's the error you're getting?

I wonder if it's because I upgraded `czifui` 5 days ago with new type files, so we need to re-run `npm i` locally to get the new type files?

Thank you!